src/test/regress/sql/multi_table_ddl.sql

```diff
@@ -42,3 +42,20 @@ SELECT * FROM pg_dist_shard_placement;
 -- check that the extension now can be dropped (and recreated)
 DROP EXTENSION citus;
 CREATE EXTENSION citus;
+
+-- create a table with a SERIAL column
+CREATE TABLE testserialtable(id serial, group_id integer);
+SELECT master_create_distributed_table('testserialtable', 'group_id', 'hash');
+SELECT master_create_worker_shards('testserialtable', 2, 1);
```

Comment:
Can you add a test documenting that so far we don't support adding serial columns later, nor ALTER SEQUENCE ... OWNED BY?